libcloud/test/compute/test_dimensiondata.py

```diff
@@ -237,6 +238,110 @@ def test_create_node_no_network(self):
                                     ex_network=None,
                                     ex_isStarted=False)
 
+    def test_create_node_mcp1_ipv4(self):
+        rootPw = NodeAuthPassword('pass123')
+        image = self.driver.list_images()[0]
+        node = self.driver.create_node(name='test2',
+                                       image=image,
+                                       auth=rootPw,
+                                       ex_description='test2 node',
+                                       ex_primary_ipv4='10.0.0.1',
+                                       ex_isStarted=False)
```

Comment:
Because it's just passing kwargs and ex_is_started is defaulted.  I will fix, but until we have a really good mocking system and xml per validation for all values in our case, this will never be caught.